README.md

```diff
@@ -38,6 +43,92 @@ subsembly
 
 Main types and API entries are defined in `runtime` folder. `runtime.ts` file in `runtime` folder defines types and constants for the frame modules and pallets.
 
+## Subsembly CLI
+
+`subsembly` is a cli tool that is used to ease the devopment of Subsembly runtimes.
+
+### Installation
+
+To install `subsembly`, run:
+
+`npm install -g subsembly`
+
+Requirements:
+- Node >= 14.0.0
+
+### Tests
+
+To run tests for the cli:
+
+- `yarn install && cd cli` - Install dependencies and go to cli directory
+
+- `npm install && npm run build && npm link` - Install dependencies, run the cli and link it globally
+
+- `npm run test` - perform tests
```

Comment:
I think that this should be placed after the documentation for the Commands and I think that it would be better to be `Subsembly CLI Tests`